k8s/src/main/scala/io/buoyant/k8s/Stabilize.scala

```diff
@@ -0,0 +1,60 @@
+package io.buoyant.k8s
+
+import com.twitter.util.{Activity, Updatable, Var}
+
+trait Stabilize {
```

Comment:
okay, I can change it – for my education, are there any particular reasons why this is considered harmful?